package.json

```
@@ -4,64 +4,50 @@
   "version": "1.0.0",
   "license": "Apache-2.0",
   "scripts": {
-    "start": "if-env NODE_ENV=production && npm run -s serve || npm run -s dev",
-    "build": "preact build --no-prerender",
-    "serve": "preact build --no-prerender && preact serve",
+    "build": "preact build",
+    "serve": "sirv build --port 8080 --cors --single",
     "dev": "preact watch",
     "lint": "eslint src",
-    "test": "jest ./tests",
-    "deploy": "gulp deploy",
-    "license": "license-checker --csv --production > licenses.csv"
+    "test": "jest"
   },
   "eslintConfig": {
-    "extends": "eslint-config-synacor"
+    "extends": "preact",
+    "ignorePatterns": [
+      "build/"
+    ]
   },
-  "eslintIgnore": [
-    "build/*"
-  ],
   "devDependencies": {
```

I think we still want the gulp dependencies for the `deploy` task (I had to bump to the latest gulp version to get it to run - the older version gave me `ReferenceError: primordials is not defined`).
```suggestion
  "devDependencies": {
    "gulp": "^4.0.2",
    "gulp-gh-pages": "^0.5.4",
```
After adding these dependencies back in, the `deploy` task seemed to run (though this is blocked by `npm run build` not working for me yet):
```
$ npm run deploy

> architecture-viewer@1.0.0 deploy /Users/dj/Code/architecture-viewer-swaraj
> gulp deploy

[22:52:45] Using gulpfile ~/Code/architecture-viewer-swaraj/gulpfile.js
[22:52:45] Starting 'deploy'...
[22:52:45] [gh-pages] No files in the stream.
[22:52:45] Finished 'deploy' after 17 ms
```